models/tokens/bnb/tokens_bnb_bep20.sql

```diff
@@ -1,5538 +1,5537 @@
-,('0x7f559adb8e2690c9b9ed8a8eabe452631d0179e2', 'ARC', 8)
-,('0x3b98bbefe14b98000f10124ca95ed298ac9db3ff', 'AceD', 18)
-,('0x3184282c417d9cdd0f07206ce9d1259c0aa0c931', 'FORB', 9)
-,('0xc5b87ecc8c044f84a0f479a91b808e3c06799811', 'S7', 18)
-,('0x74c609b16512869b1873f5a9d7999deee386e740', 'SANTASHIB', 9)
-,('0x9eb5b7902d2be0b5aaba2f096e043d3cd804e6df', 'MPS', 18)
-,('0x14ace3d3429408bfa8562099a253172913ad71bd', 'DCH', 18)
-,('0x0ead9d6bda8408cc34e938ea35559db738a7c76a', 'MCAT', 9)
-,('0xd80bea63a208770e1c371dfbf70cb13469d29ae6', 'xAGC', 9)
-,('0xc0b943b63e605cf8a75c7832c2baf46629f7f762', 'TREASURE', 18)
-,('0x239ec95667e37929d33066a8df8ddc9444dbcbca', 'DFI', 18)
-,('0xb110961c579d41a023e03dd91f7712101187e9be', 'HMA', 18)
-,('0x45e55e756496c046074af00709e17183c220c438', 'DEX', 18)
-,('0xc3b538219b73efd57b75f616e909a728be945bbb', 'EMERALD', 18)
-,('0x93d3397d08676246425ff016c8180322e334c2b2', 'KLP', 18)
-,('0x8d947e7c2ce0ed24a801b2d23bff72d933c842a5', 'NFTPUNK2.0', 18)
-,('0xe86752f7655b61161d6b71987ebc9e4f4f5eead7', 'AMT', 18)
-,('0xcb15fc54d0655150fa3784ec4a355b76a951e4cf', 'SquidPet', 9)
-,('0xe7cb24f449973d5b3520e5b93d88b405903c75fb', 'BNBTC', 8)
-,('0x0af45ae58cdb1aeaeeb2e2c4af53158513b89206', 'veRabbit', 18)
-,('0xa25fc408ef05321103243557c851101f9acee608', 'CHIP', 18)
-,('0x2f7694695d71caaae29fdf3f140530d057de970b', 'xDRACE', 18)
-,('0x10964c2ffdea1e99b5e26d102516d9b03368915f', 'RIPH', 18)
-,('0x1e4d9722a7be03876c121285e9638acb88a52b65', 'KRC', 18)
-,('0x55b53855eae06c4744841dbfa06fce335db4355b', 'SSB', 9)
-,('0x58992eb2743e738073c62f52b13b6919880baa43', 'VCOIN', 18)
-,('0x379f29911956052899e09233a3914f4e1eca075c', 'Meta', 18)
-,('0x84f4f7cdb4574c9556a494dab18ffc1d1d22316c', 'KINGSHIB', 9)
-,('0x1f64fdad335ed784898effb5ce22d54d8f432523', '10SET', 18)
-,('0xeb0bdd8f51c867d787f0fc6b2dee360ca31628e3', 'NX', 18)
-,('0xa5360c2070faecfc231fd6bd743fe88382f2991d', 'SAFELIGHT', 9)
-,('0xaad87f47cdea777faf87e7602e91e3a6afbe4d57', 'PYE', 18)
-,('0x54d625b45bcb1326f64ce79fac313f4d8f47ae24', 'CROP', 9)
-,('0xc77dd3ade7b717583e0924466e4e474a5673332c', 'BSTS', 9)
-,('0x5067c6e9e6c443372f2e62946273abbf3cc2f2b3', 'FIBO', 9)
-,('0x0d499b25bce7aa72bb6c50e434e2ed26fe1e785d', 'BRT', 18)
-,('0x89366229ad6aace698d1454dbf1323cb45eb575f', 'METAMONKEY', 18)
-,('0x118f073796821da3e9901061b05c0b36377b877e', 'CATE', 8)
-,('0x339fe932809e39a95b621a7f88bbf6c08eb6c978', 'TXBIT', 18)
-,('0xe49bfc53a195a62d78a941a1967d7b0f83a47c14', 'TAC', 9)
-,('0xd6b8dcc3466d2045b4153ba717805724cecbf788', 'dubai', 18)
-,('0x330540a9d998442dcbc396165d3ddc5052077bb1', 'FLOKI', 9)
-,('0xba53af4c2f1649f82e8070fb306ddbf2771a1950', 'moo1INCH1INCH', 18)
-,('0xd308af46aba59ec24b515951000d857984345168', 'SJM', 9)
-,('0xcc4861adc85d86ae8c9cc9d19f4f109969eb4f7b', 'DOG', 18)
-,('0xcf958b53ec9340886d72bb4f5f2977e8c2ab64d3', 'GLB', 18)
-,('0x78a18db278f9c7c9657f61da519e6ef43794dd5d', 'SPI', 18)
-,('0x9aa6fc71aed1130dee06a91a487bf5ea481de80d', 'COCO', 18)
-,('0xe6929a329bf
```

Comment:
looks like one gets excluded here too? maybe an issue with the automated script?

Reply:
after seeing in first two blockcahins, i'll just assume it could happen to all?